rcl/src/rcl/timer.c

```diff
@@ -22,7 +22,6 @@ extern "C"
 #include <inttypes.h>
 
 #include "rcl/error_handling.h"
-#include "rcutils/logging_macros.h"
```

Comment:
Why the removal of this include?  Those macros are used in the file.

Reply:
Done.